src/middleware/validateSchema.js

```diff
@@ -3,16 +3,15 @@
 const {
 	isBoolean, includes, map, has, get
 } = require('lodash')
-const { schemas } = require('../lib')
+const { schemas } = require('../schema')
```

Comment:
the import is wrong @beblicarl. kindly double check

Reply:
alright, i am working on it